demos/form/templates/input.pug

```diff
@@ -1,25 +1,25 @@
 | {LabeledGroup}
-div(class="{$field_class} field atk-form-group")
+div(class="{$control_class} field atk-form-group")
     label {$label}
     div(class="{$width} {$class} fields")
         | {$Content}
     | {$Hint}
 | {/}
 
 | {NoLabelGroup}
-div(class="{$field_class} field atk-form-group")
+div(class="{$control_class} field atk-form-group")
     div(class="{$width} {$class} fields")
         | {$Content}
 | {/}
 
-| {InputField}
-div(class="{$field_class} field")
+| {LabeledControl}
```

Comment:
is this for general control or input only?

Reply:
This is for the control so I named it accordingly.
Form control (in ATK) = label + input + hint + validation message

Reply:
I agree that input is subpart of control. But if this is related strictly with input, I think it is more logical to keep input name here.

Reply:
It is not strictly with input as I wrote.
LabelledControl = label + input + hint + validation message
NoLabelControl = input + hint + validation message
Input = input (this is the region for the pure input)